batchflow/models/torch/base.py

```diff
@@ -316,6 +316,8 @@ def __init__(self, config=None):
         self.devices = []
 
         # Train procedure and ifrastructure
+        self.kaiming_normal_init = None
```

Comment:
Can you move `init` to a separate PR? I reckon it would be merged a touch faster, than the tutorial

Reply:
What about other way of applying initialization strategies: a key `init` in the config, that can be used to pass a string (e.g. `default`) to choose a pre-defined initialization strategy (a callable on nn.Module), or a callable that will be applied directly?

Reply:
It is a good idea. Also, this PR contains some fixes and additions; should they be separated too?